test/snoowrap.spec.js

```diff
@@ -1079,6 +1079,34 @@ describe('snoowrap', function () {
       expect(list3).to.have.lengthOf(2);
       expect(list3.every(post => post instanceof snoowrap.objects.Submission)).to.be.true();
     });
+    it('can get multiple posts by fullname id', async () => {
+      const posts = await r.getContentByIds(['t3_9l9vof', 't3_9la341']);
+      expect(posts).to.have.lengthOf(2);
+      expect(posts.every(post => post instanceof snoowrap.objects.Submission)).to.be.true();
+    });
+    it('can get multiple posts by submission object', async () => {
+      const posts = await r.getContentByIds([r.getSubmission('9l9vof'), r.getSubmission('9la341')]);
+      expect(posts).to.have.lengthOf(2);
+      expect(posts.every(post => post instanceof snoowrap.objects.Submission)).to.be.true();
+    });
+    it('can get multiple comments by fullname id', async () => {
+      const comments = await r.getContentByIds(['t1_erx7rym', 't1_erx7tl8']);
+      expect(comments).to.have.lengthOf(2);
+      expect(comments.every(comment => comment instanceof snoowrap.objects.Comment)).to.be.true();
+    });
+    it('can get multiple comments by comment object', async () => {
```

Comment:
I'd merge this with get submissions. Get one of each to see if it works. Could also combine it with a string to reduce the tests once more